sdk-setup/src/mb2

```diff
@@ -1194,6 +1261,84 @@ run_prepare() {
     return $rc
 }
 
+cmake_enable_host_paths() {
```

Comment:
Could we use some more generic name for the two functions? Something that could be understood just that some unspecified setup/cleanup is done for cmake.